### PR TITLE
[optimize](show) Support show statement info when execute show proc '/current_query_stmts' statement

### LINKED ISSUE #7488: [Enhancement] Support show statement info when execute show proc '/current_queries' statement
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

Now, it is not convenient for user to see what current sql are running when execute show proc '/current_queries' statement

### Solution

Support show statement info when execute show proc '/current_query_stmts' statement

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Close related #issue (replace it with issue number if it exists).

Describe the overview of changes, and introduce why we need it.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [x] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #7488 ) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowProcStmt.java [hunk 1]
```diff
@@ -46,14 +46,9 @@ public ProcNodeInterface getNode() {
...
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
         }
-
         node = ProcService.getInstance().open(path);
-        if (node == null) {
-            ErrorReport.reportAnalysisException(ErrorCode.ERR_WRONG_PROC_PATH, path);
-        }
     }
 
     @Override
```
Comment (jackwener) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowProcStmt.java:54]: Could you replace code line 56-58 by the way, it includes redundant code.
```java
StringBuilder sb = new StringBuilder();
        sb.append("SHOW PROC '").append(path).append("'");
        return sb.toString();
```
with 

```java
        return "SHOW PROC '" + path + "'";
```

**Comment by morningman** [comment]
I am afraid that this could make the result of `show proc "/current_queries"` mess-up. Because the SQL string may be very long or poor formatting.

Why not using `show proc "/current_queries/query_id"` to see the SQL stmt?

**Comment by caiconghui** [comment]
> I am afraid that this could make the result of `show proc "/current_queries"` mess-up. Because the SQL string may be very long or poor formatting.
> 
> Why not using `show proc "/current_queries/query_id"` to see the SQL stmt?

In fact, in most cases, we want to see which users are executing which statements through show proc'/current_queries'. In order to quickly troubleshoot some problems, it is not convenient to query id indirectly through query id. We need to find out all query id first. Then when you look for stamtement, it may not be there. Of course, the problem you mentioned exists.

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowProcStmt.java (hunk 1 above)
Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowProcStmt.java:54]: @jackwener hi, I think both ways of coding can be ok. There is no clear specification when to use ‘+’ and when to use StringBuilder explicitly.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**